pkg/sql/parser/eval_test.go

```diff
@@ -606,7 +606,7 @@ func TestEval(t *testing.T) {
 		{`CAST('123' AS int) + 1`, `124`},
 		{`CAST(NULL AS int)`, `NULL`},
 		{`'hello'::char(2)`, `'he'`},
-		{`'hello'::bytes`, `b'hello'`},
+		{`'hello'::bytes`, `b'\x68\x65\x6c\x6c\x6f'`},
```

Comment:
Like we discussed I think it would be better to have this stringify to '\x68656c6c6f' for pg compatibility.